apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2291,6 +2293,23 @@ const uploadConversationsWithFoldersEpic: AppEpic = (action$) =>
                 conversations: conversations,
               }),
             ),
+            iif(
+              () => !!payload.selectFirst,
+              concat(
+                of(
+                  ConversationsActions.selectConversations({
+                    conversationIds: [conversations[0].id],
+                  }),
+                ),
+                of(
+                  UIActions.openFolder({
+                    featureType: FeatureType.Chat,
+                    id: conversations[0].folderId,
+                  }),
```

Comment:
this action will upload and open only one folder, but chat can be several folders
e.g. if `folder1/folder2/folder3/chat1` then you open only `folder3`